### ISSUE TITLE
[Question/Feedback]: 

### ISSUE THREAD

**Issue by ahelland** [Azure/Azure-Verified-Modules#1470]
### Check for previous/existing GitHub issues

- [X] I have checked for previous/existing GitHub issues

### Description

Currently .NET Aspire supports provisioning Azure resources for development-time usage:
https://github.com/dotnet/aspire/tree/main/src/Aspire.Hosting.Azure

You create Bicep files like you normally do, inject params & use the outputs. For secrets you can employ some secret sauce where you add a Key Vault Secret within your Bicep and Aspire creates a Vault and stuffs it in there so you don't risk checking in things you shouldn't.

As it's plain Bicep you can of course add Verified Modules as you see fit within, but that's outside Aspire's scope. For deployment-time you're driven towards using the Azure Developer CLI. (Which isn't using AVM either, but that's a separate thread: https://github.com/Azure/azure-dev/issues/2386)

I think it would be useful to be able to skip straight to consuming modules from Aspire code like 
`var foo = builder.AddBicepVerifiedModule(...)`

On a technical level it would probably be useful to support modules in a generic way, but after all this is the AVM repo :)

Does this make sense or should it be filed over in the Aspire repo instead? Or is it a matter of "if the community wants it; build it"?

**Comment by matebarabas** [comment]
@Azure/avm-core-team-technical-bicep, can you please help with this question?

**Comment by MisinformedDNA** [comment]
https://github.com/microsoft/aspire/issues/13499

**Comment by jtracey93** [comment]
Yeah sorry for the delay here. This should be raised on the aspire repo and we are happy to discuss and support over there on the issue linked https://github.com/microsoft/aspire/issues/13499
